.circleci/config.yml

```diff
@@ -48,6 +48,12 @@ commands:
       - run:
           name: CI (Jaeger)
           command: "cd exporters/jaeger && bundle exec rake"
+      - run:
+          name: Bundle (OpenTracing Bridge)
+          command: "cd bridges/opentracing && gem install --no-document bundler && bundle install --jobs=3 --retry=3"
```

Comment:
Very weird jruby thing where if you bundle OTrace bridge before CI jaeger (or vise versa) whichever you bundled before the other one, fails its CI step with e.g.
```
File does not exist: thrift
```